string current_state
string target_state
---
bool is_transition_successful